client/src/ui/Admin.jsx

```diff
@@ -39,69 +40,45 @@ export class Admin extends Component {
       // 1: database professor clearing function was called, scraper is running
       // 2: database professor clearing function has completed
       adminPanelHTML: "Invalid Credentials"
-    }
-
-    // define bart alert message constants
-    Bert.defaults = {
-      hideDelay: 8000, //time alert stays on screen
-      type: 'danger',
-      style: 'growl-top-right',
-      icon: 'fa-warning',
     };
 
     this.approveReview.bind(this);
     this.removeReview.bind(this);
     this.unReportReview.bind(this);
   }
-  //
-  // componentWillMount(){
-  //   if(Session.get("token") != undefined){
-  //     Meteor.call('tokenIsAdmin', Session.get("token"), (error, result) => {
-  //       if (!error && result === true) {
-  //         setTimeout(function () {
-  //                 console.log("here0");
-  //         }, 3000);
-  //             adminPanelHTML  = (
-  //
-  //             )
-  //             this.setState({adminPanelHTML: adminPanelHTML})
-  //       }
-  //     })
-  //   }
-  // }
 
   // Call when user asks to approve a review. Accesses the Reviews database
   // and changes the review with this id to visible.
   approveReview(review) {
-    Meteor.call('makeVisible', review, Session.get("token"), (error, result) => {
+    Meteor.call("makeVisible", review, Session.get("token"), (error, result) => {
       if (!error && result === 1) {
-        console.log("Review approved")
+        console.log("Review approved");
       } else {
-        console.log(error)
+        console.log(error);
       }
     });
   }
 
   // Call when user asks to remove a review. Accesses the Reviews database
   // and deletes the review with this id.
   removeReview(review) {
-    Meteor.call('removeReview', review, Session.get("token"), (error, result) => {
+    Meteor.call("removeReview", review, Session.get("token"), (error, result) => {
       if (!error && result === 1) {
-        console.log("Review removed")
+        console.log("Review removed");
       } else {
-        console.log(error)
+        console.log(error);
       }
     });
   }
 
   // Call when user asks to un-report a reported review. Accesses the Reviews database
   // and changes the reported flag for this review to false.
   unReportReview(review) {
-    Meteor.call('undoReportReview', review, Session.get("token"), (error, result) => {
+    Meteor.call("undoReportReview", review, Session.get("token"), (error, result) => {
       if (!error && result === 1) {
-        console.log("Review unreported")
+        console.log("Review unreported");
       } else {
-        console.log(error)
+        console.log(error);
```

Comment:
For the sake of reducing merge conflicts, could you revert these and other similar whitespace and style only changes?
I understand that not having a consistent style in the repo is not a good thing, but let's defer that until we have all hanging changes merged in, lock down master, and perform a complete code style cleanup.

Reply:
Ugh, I must have reflexively hit the format command at some point.